recipes/pyprojroot/meta.yaml

```
@@ -0,0 +1,38 @@
+{% set name = "pyprojroot" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "4849c4e11beef9e66631f5cd77868b8ee7ecf6f3cedcb363104314613a1d24cf"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```

Could it be ``noarch``?
```suggestion
  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
  noarch: python
```